static/app/views/dashboardsV2/widgetBuilder/utils.tsx

```diff
@@ -92,8 +92,7 @@ export function normalizeQueries(
   }
 
   if ([DisplayType.TABLE, DisplayType.TOP_N].includes(displayType)) {
-    // Reset orderby
-    return queries.map(query => ({...query, orderby: ''}));
+    return queries;
```

Comment:
needs tests for checking this 😭

Reply:
@dashed I'm making a separate PR to add test coverage (like literally rn)

Reply:
@shruthilayaj thanks 🌮